test/test_lib/utils.js

```diff
@@ -58,6 +57,8 @@ function Utils() {
 
 Utils.prototype = {
 
+    this: this,
```

Comment:
Why is this needed?

Reply:
Not needed, removed ➖ 
Was fighting with JS prototype logic 🤼‍♂️